src/views/clusteroverview/SettingsTab/ClusterTab/components/EnablePreviewFeaturesSection/EnablePreviewFeaturesSection.tsx

```diff
@@ -10,14 +11,32 @@ import { Stack, StackItem } from '@patternfly/react-core';
 
 import ExpandSection from '../../../ExpandSection/ExpandSection';
 
+import PreviewFeaturesPopover from './PreviewFeaturesPopover';
+
 const EnablePreviewFeaturesSection: FC = () => {
   const { t } = useKubevirtTranslation();
 
   return (
-    <ExpandSection toggleText={t('Preview features')}>
+    <ExpandSection
+      toggleContent={
+        <>
+          {t('Preview features')}
+          <PreviewFeaturesPopover />
+        </>
+      }
+    >
       <Stack hasGutter>
         <StackItem isFilled>
           <EnableFeatureCheckbox
+            externalLink="https://kubevirt.io/user-guide/operations/hotplug_interfaces/#hotplug-network-interfaces"
+            featureName={BRIDGED_NIC_HOTPLUG_ENABLED}
+            id={BRIDGED_NIC_HOTPLUG_ENABLED}
+            label={t('Enable bridged network interface hot plug')}
+          />
+        </StackItem>
+        <StackItem isFilled>
+          <EnableFeatureCheckbox
+            externalLink="https://kubevirt.io/user-guide/virtual_machines/instancetypes/"
```

Comment:
I think this URL is also used elsewhere in the app, please check and if not existing as a const, create one for this